lib/metasploit/framework/hashes.rb

```diff
@@ -0,0 +1,129 @@
+module Metasploit
+  module Framework
+    # This module contains utilities for hashes, including one to identify them
+    # Resource list:
+    #  https://code.google.com/archive/p/hash-identifier/
+    #  https://github.com/psypanda/hashID
+    #  https://hashcat.net/wiki/doku.php?id=example_hashes
+    #  http://pentestmonkey.net/cheat-sheet/john-the-ripper-hash-formats
+    #  https://openwall.info/wiki/john/sample-hashes
+    #  QNX formats -> https://moar.so/blog/qnx-password-hash-formats.html
+    # rubocop:disable Metrics/ModuleLength
+    module Hashes
+      JTR_NTLMV1 = 'netntlm'.freeze
+      JTR_NTLMV2 = 'netntlmv2'.freeze
+      def identify_hash(hash)
```

Comment:
This should address the spec failure and is a valid change needed for this to work as a utility module.
```suggestion
      def self.identify_hash(hash)
```